EcoSoapBank/Helpers/UIKit/UIView+Layout.swift

```diff
@@ -9,7 +9,12 @@
 import UIKit
 
 extension UIView {
-    
+
+    /// Applies constraints to all sides of superview with constant padding defined by the provided insets.
+    ///
+    /// **REQUIRED PRECONDITIONS**:
+    /// - View must be embedded in a superview
+    /// - `translatesAutoresizingMaskIntoConstraints` must be false
     func fillSuperview(withPadding padding: UIEdgeInsets = .zero) {
         assert(translatesAutoresizingMaskIntoConstraints == false,
```

Comment:
I also considered just setting this to false here, but I figured that was maybe a bit redundant considering the convenience method I made for adding subviews using auto layout.